### DISCUSSION TITLE
Central Meme Database To filter meme faces?

### DISCUSSION THREAD

**Discussion by JustSuperHuman** [immich-app/immich#9605] [General]
I know we all have the problem with screenshots and memes. 

I'm realizing I have so many meme faces in the face detection - I wonder if there could be some central meme repo so we can filter them all out or help Immich identify "That's a meme" 

Probably a side project more than core Immich.

**Comment by auanasgheps** [comment]
It may seem a a "silly" question, but it's not! 

I have a similar case when I take photos of TV during news, advertising, newspapers and so on,  where politicians or famous people are pictured. I don't care to tag politicians or similar people in my photos :) 

So my request extends to famous/know people. If there's a database we can plug in to, would be great for the end user to choose which categories of faces to exclude/filter.

**Comment by multi-suggester** [comment]
i think you may consider writing "meme" at the search bar and might get some good results? , i cant confirm it locally at the moment though
